pkgs/misc/emulators/wine/base.nix

```diff
@@ -103,6 +103,8 @@ stdenv.mkDerivation ((lib.optionalAttrs (! isNull buildScript) {
   
   enableParallelBuilding = true;
 
+  hardeningDisable = [ "bindnow" ];
```

Comment:
Can you add a comment here, linking to the Nixpkgs & WineHQ bugs.

Reply:
Done.